recipes/eva-sub-cli/build.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+VCF_VALIDATOR_VERSION=0.9.6
+BIOVALIDATOR_VERSION=2.1.0
+EVA_PYUTILS_VERSION=0.6.1
+
+EVA_SUB_CLI="${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}"
+mkdir -p ${PREFIX}/bin ${EVA_SUB_CLI}
+
+# Install eva-sub-cli
+$PYTHON -m pip install .
+cp bin/* ${PREFIX}/bin
+echo "Done with eva-sub-cli"
+
+cd ${EVA_SUB_CLI}
+
+# Install python dependencies not yet on conda
+curl -Lo eva-pyutils.zip https://github.com/EBIvariation/eva-common-pyutils/archive/refs/tags/v${EVA_PYUTILS_VERSION}.zip \
+    && unzip -q eva-pyutils.zip && rm eva-pyutils.zip \
+    && cd eva-common-pyutils-${EVA_PYUTILS_VERSION} \
+    && $PYTHON setup.py install \
+    && cd ..
+
+# Install biovalidator from source
+# Includes some workarounds that can be cleaned up once a new version is released
+curl -Lo biovalidator.zip https://github.com/elixir-europe/biovalidator/archive/refs/tags/v${BIOVALIDATOR_VERSION}.zip \
+    && unzip -q biovalidator.zip && rm biovalidator.zip \
+    && cd biovalidator-${BIOVALIDATOR_VERSION} \
+    && bash -c "cat <(echo '#!/usr/bin/env node') <(cat src/biovalidator.js) > tmp" \
+    && mv tmp src/biovalidator.js \
+    && chmod +x src/biovalidator.js \
+    && sed -i 's/dist/src/' package.json \
+    && npm install && npm install -g \
+    && cd ..
+echo "Done with biovalidator"
+
+# Download pre-built vcf-validator
+# Check if linux or osx
+if [ -z ${OSX_ARCH+x} ]; then
```

Comment:
Why can we not depend on a conda package here? Or why do we need a pre-build binary?

Reply:
Thanks for your review. The vcf-validator build is a bit flaky (see [here](https://github.com/EBIvariation/vcf-validator/blob/master/install_dependencies.sh)), and so we prefer to build vcf-validator as a static binary with all its dependencies, in a controlled environment where we can continuously test (on [Github actions](https://github.com/EBIvariation/vcf-validator/blob/master/.github/workflows/build.yml)).

We're happy to provide it as a separate conda recipe, but it would likely still need to be downloaded pre-built in that recipe. If that's not possible we'll need to look into it more thoroughly.

Reply:
now relies on the conda package